examples/SolverComparer.cpp

```diff
@@ -228,6 +234,10 @@ int main(int argc, char *argv[]) {
     runPerturb();
   else if(stats)
     runStats();
+#ifdef gtsam_use_tbb
```

Comment:
Should be upper cased?

Reply:
yikes, thx for catching that. Fixed it up.

Reply:
I am surprised that it didn't trigger CI fails (or I missed it?)

Reply:
No it passed, im p sure preprocessor directives can be lowercase, just highly frowned upon since it can much more easily conflict with a variable name.

Reply:
I mean in your previous file I see two types of macros, one in upper case and one in smaller case.

Reply:
That is not triggering a CI fail because by default the smaller case is not defined, *HOWEVER* this means that our test suite is not catching that that piece of code is missing.